homeassistant/components/nextcloud/__init__.py

```diff
@@ -1,79 +1,28 @@
 """The Nextcloud integration."""
-import logging
 
 from nextcloudmonitor import (
     NextcloudMonitor,
     NextcloudMonitorAuthorizationError,
     NextcloudMonitorConnectionError,
     NextcloudMonitorRequestError,
 )
-import voluptuous as vol
 
-from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
     CONF_PASSWORD,
-    CONF_SCAN_INTERVAL,
     CONF_URL,
     CONF_USERNAME,
     CONF_VERIFY_SSL,
     Platform,
 )
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
-from homeassistant.helpers import config_validation as cv
-from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
-from homeassistant.helpers.typing import ConfigType
 
-from .const import DEFAULT_SCAN_INTERVAL, DOMAIN
+from .const import DOMAIN
 from .coordinator import NextcloudDataUpdateCoordinator
 
-_LOGGER = logging.getLogger(__name__)
 PLATFORMS = (Platform.SENSOR, Platform.BINARY_SENSOR)
 
-# Validate user configuration
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We need to keep the config schema around for backward compatibility

```py
CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Reply:
What is the reason?

Reply:
> for backward compatibility